src/components/ProductList/ProductList.jsx

```diff
@@ -4,7 +4,23 @@ import { getAllProducts } from "../../helpers/getAllProducts";
 import { SortData } from "../../helpers/SortData";
 import { FilterData } from "../../helpers/FilterData";
 import { ProductCard } from "../Card/ProductCard";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
 export const ProductList = (props) => {
+  const { cart, cartDispatch } = useCart();
+  const { wishlist, wishlistDispatch } = useWishlist();
+  const onAddToCart = (product) => {
+    cartDispatch({ type: "ADD", payload: product });
+  };
+  const onAddToWishlist = (product) => {
```

Comment:
can remove the parentheses around the argument and the curly braces incase of one line arrow functions.